glusterfs/tests/docker/docker-compose.yaml

```diff
@@ -0,0 +1,11 @@
+version: '2'
+services:
+    gs1:
+        container_name: gluster-node-1
+        image: gluster/gluster-centos:latest
```

Comment:
Should we pin the version?

Reply:
we could pin the version, but it doesn't seem to be updated frequently. latest is from 1+ years ago.